src/quo/components/notifications/activity_log/view.cljs

```diff
@@ -9,11 +9,10 @@
     [quo.components.tags.status-tags :as status-tags]
     [quo.foundations.colors :as colors]
     [react-native.core :as rn]
-    [reagent.core :as reagent]
     [utils.i18n :as i18n]))
 
 (defn- activity-reply-text-input
-  [{:keys [on-update-reply max-reply-length valid-reply?]} reply-input]
+  [{:keys [on-update-reply max-reply-length valid-reply?]} reply-input set-reply-input]
```

Comment:
why can't we just use the single map?
It's more consitent with react style.

`[props]`
and for 2 params it's generally used as
`[props children]`

Reply:
i don't want to change logic much, just replacing atoms with state

Reply:
it's not really logic but anyway,  im trying to minimize changes

Reply:
it's kind of a minimal change tbh. Also you can leave `reply-input` as is but just assoc `set-reply-input` into props below 👍